examples/advanced/quantum_volume.py

```diff
@@ -68,7 +68,7 @@ def parse_arguments(args):
                         type=int,
                         help='Seed for the Random Number Generator.')
     parser.add_argument(
-        '--num_repetitions',
+        '--num_circuits',
         default=100,
         type=int,
         help='The number of times to run the circuit on the quantum computer.'
```

Comment:
This doc should now say something like "The number of circuits to generate". Or, to not reproduce the message, "Pass-through to the Quantum Volume module".